migtests/tests/pg/multiple-schemas/validate

```diff
@@ -145,16 +146,27 @@ def migration_completed_checks(tgt):
 			"orafce", "pageinspect", "pg_buffercache", "pg_cron", "pg_freespacemap", "pg_hint_plan", "pg_prewarm", "pg_stat_monitor",
 			"pg_stat_statements", "pg_trgm", "pg_visibility", "pgaudit", "pgcrypto", "pgrowlocks", "pgstattuple", "plpgsql",
 			"postgres_fdw", "refint", "seg", "sslinfo", "tablefunc", "tcn", "timetravel", "tsm_system_rows",
-			"tsm_system_time", "unaccent", "uuid-ossp", "yb_pg_metrics", "yb_test_extension" ]
+			"tsm_system_time", "unaccent", "uuid-ossp", "yb_pg_metrics", "yb_test_extension", "vector" ]
 		extensions = tgt.get_available_extensions()
 		print(f"expected extensions: {expectedExtensions}")
 		print(f"reported extensions: {extensions}")
 		tgtVersion = tgt.get_target_version()
 		print(f"target version: {tgtVersion}")
-		if "2.16" in tgtVersion:
-			expectedExtensions.remove("hll")
+		if "2.25" in tgtVersion:
+			expectedExtensions.remove("timetravel")
+			expectedExtensions.extend(["pg_partman", "pg_walinspect", "old_snapshot", "pgtap", "yb_ycql_utils", "pg_surgery", "yb_xcluster_ddl_replication"])
+		if "2024.2" in tgtVersion:
+			expectedExtensions.extend(["pg_partman", "pgtap", "yb_ycql_utils", "yb_xcluster_ddl_replication"])
+		if "2024.1" in tgtVersion:
+			expectedExtensions.extend(["yb_ycql_utils"])
 		for extension in expectedExtensions:
 			assert extension in extensions, f"expected extension is not reported: {extension}"
+		
+		# Check for extra/unexpected extensions
+		extra_extensions = set(extensions) - set(expectedExtensions)
```

Comment:
nice 👍 
in general, it might be worth moving this test to the golang unit tests which we run against each version. that way, we will immediately catch it. 

Here, in these tests, we will only catch when we actually run our test pipeline against that target-db-version. 

cc: @sanyamsinghal

Reply:
Makes sense. Let me know if there's an action item on me for this.